lib/reporter.js

```diff
@@ -37,6 +38,13 @@ const getOsArch = function () {
 
 module.exports =
 class Reporter {
+  static incrementCounter (counterName) {
+    store.incrementCounter(counterName)
+  }
+
+  static addCustomEvent (event, eventName) {
+    store.addCustomEvent(event, eventName)
+  }
   static sendEvent (category, action, label, value) {
```

Comment:
I'm guessing that you'd want anyone writing new code to use `addCustomEvent` instead of `sendEvent`. Is that correct? If so, I'd recommend marking `sendEvent` as deprecated [[example](https://github.com/atom/atom/blob/bee32d58d9b73d7eff70b83086c9a18033ea46aa/src/text-editor-registry.js#L179)]. What do you think?

Reply:
Also, could/should it be called `sendCustomEvent`?  To me, `add` sounds like we're registering an event type rather than sending some information

Reply:
ah, good call.  Will change this too.